### ISSUE TITLE
Resolve Lighthouse Issues

### ISSUE THREAD

**Issue by maxwellward** [ecadlabs/taquito-test-dapp-vue#58]
There are a number of performance and accessibility issues detected by Lighthouse. These should be resolved as best as possible.

https://pagespeed.web.dev/analysis/https-ghostnet-dapp-taquito-io/6qx1u1ddy8?form_factor=desktop
